.github/workflows/image-release.yaml

```diff
@@ -160,7 +173,7 @@ jobs:
         provenance: false
         tags: |
           quay.io/${{env.REPO_ORG}}/${{env.REPO_NAME}}:${{ steps.vars.outputs.sha_short }}
-          quay.io/${{env.REPO_ORG}}/${{env.REPO_NAME}}:${{ github.event.inputs.tag }}
+          quay.io/${{env.REPO_ORG}}/${{env.REPO_NAME}}:${{ env.tag }}
 
     - name: Create tag
```

Comment:
I don't think we need to create tag anymore since we should expect the tag to already exist?

Reply:
What do you mean?